exercises/atbash-cipher/atbash_cipher_test.rb

```diff
@@ -1,46 +1,103 @@
 require 'minitest/autorun'
 require_relative 'atbash_cipher'
 
-class AtbashTest < Minitest::Test
-  def test_encode_no
-    assert_equal 'ml', Atbash.encode('no')
+# Common test data version: 1.2.0 d5238bd
+class AtbashCipherTest < Minitest::Test
+  def test_encode_yes
+    # skip
+    plaintext = 'yes'
+    ciphertext = 'bvh'
```

Comment:
For these short values using variables seems like overkill, but I assume a cleaner `generator` file is more important.